crates/yulgen/src/runtime/functions/data.rs

```diff
@@ -380,3 +384,16 @@ pub fn load_data_string() -> yul::Statement {
         }
     }
 }
+
+/// Returns a pointer to the array item at the requested index.
+/// Reverts with a panic if the index is out of bounds.
+pub fn get_array_item() -> yul::Statement {
```

Comment:
I realize that this might eventually be handled at Fe level (std lib) but it is small enough to add now and is preventing me from adding further differential contract fuzzing tests as this comes up quite frequently when fuzzing with arrays involved.